spec/suites/layer/tile/TileLayerSpec.js

```diff
@@ -430,6 +430,28 @@ describe('TileLayer', function () {
 				});
 			});
 		}
+
+		it('loads tiles with matching min/maxZoom and detectRetina', function (done) {
```

Comment:
I'd prefer more straightforward tests just ensuring that min/maxZoom is changed in expected way.

Reply:
I agree. I didn't know what the issue was until I wrote this test.